client/src/screens/UnconfiguredScreen.tsx

```diff
@@ -212,8 +212,7 @@ const UnconfiguredScreen = () => {
           centerContent
           content={
             <Prose textCenter>
-              <h1>Election successfully loaded!</h1>
-              <Loading as="h2">Generating all ballot styles</Loading>
+              <Loading as="h1">Building ballot styles</Loading>
```

Comment:
How about "Election loading" because then once modal disappears and ballots are listed is it not clear that the election has loaded?
```suggestion
              <Loading as="h1">Election loading</Loading>
```